### ISSUE TITLE
How to inspect a view that takes a closure

### ISSUE THREAD

**Issue by nalexn** [nalexn/ViewInspector#68]
Moved [discussion](https://github.com/nalexn/ViewInspector/discussions/67).

I have a view with the following 

```swift
    struct FileLinkView<LinkView:View>: View {
        var item:FBFile
        let linkView:(URL)->LinkView
        ...
    }
```

I cannot figure out how to inspect it live here is the test case but I get the error the "Generic parameter 'LinkView' cannot be inferred

```swift
    func testListPreviewFile() {
        let bundlePath = getBundleTestRoot()
        let resourceFile:URL = bundlePath.appendingPathComponent("Baymax.jpg")
        let item = FBFile(filePath: resourceFile, xInfo0: nil, xInfo1: nil)
        
        let preview = PreviewController(url: resourceFile)
        
        let view = FileLinkView(item: item) { file in
            preview}
        do {
            try view.inspect().view(FileLinkView.self)  //<- Inferred Error Occurs here
            XCTAssert(false, "View did not throw error")
        } catch {
           
        }
    }
```

Any help would be appreciated.

**Comment by nalexn** [comment]
@rlegault33

You should insert a concrete generic parameter of your view, for example, `EmptyView`:

```swift
try view.inspect().view(FileLinkView<EmptyView>.self)
```

Also, instead of writing `do { } catch { }` you can simply annotate the test function with `throws`. See the [guide](https://github.com/nalexn/ViewInspector/blob/master/guide.md#the-basics).

**Comment by rlegault33** [comment]
When I try I get the error Cannot find 'EmptyView' in scope.

    try view.inspect().view(FileLinkView<EmptyView>.self) < == error Cannot find 'EmptyView' in scope.

Thanks for the throw suggestion. I was using that but took it out temporarily to simplify the above code to one line that needed fixing.

**Comment by nalexn** [comment]
Try adding `import SwiftUI`

**Comment by rlegault33** [comment]
That did it.
Thanks.

add:

     import SwiftUI

change line to

    try view.inspect().view(FileLinkView<EmptyView>.self)
